Quadriga/src/main/java/edu/asu/spring/quadriga/web/settings/WebsiteAboutEditController.java

```diff
@@ -150,21 +144,19 @@ public ModelAndView saveAbout(@ProjectIdentifier @PathVariable("ProjectId") Stri
     @InjectProjectById
     public ResponseEntity<String> visualizeNetworks(@ProjectIdentifier @PathVariable("projectId") String projectid,
             @PathVariable("networkId") String networkId, Principal principal,
-            @CheckAccess @InjectProject IProject project) {
-        ITransformedNetwork transformedNetwork = null;
+            @CheckAccess @InjectProject IProject project) throws QuadrigaStorageException {
+        boolean isValid = false;
         try {
-            transformedNetwork = transformationManager.getTransformedNetwork(networkId);
-        } catch (QuadrigaStorageException qse) {
-            logger.error("Error while retrieving networks for display:", qse);
+            isValid = nwManager.getNetworkIdsInProject(project.getProjectId()).contains(networkId);
```

Comment:
instead of having to get all network ids in a project (which involves at least 2 iterations), you could just ask the network for its project (you just have to call a few ```get...``` methods).